src/01.UI-Elements/ListItem.test.js

```diff
@@ -4,77 +4,69 @@ import user from '@testing-library/user-event'
-    const { getByText } = render(<ListItem title="Butter" />)
-
-    expect(getByText('Butter')).toBeInTheDocument()
+    const { getByTestId } = render(<ListItem title="Butter" />)
+    const title = getByTestId('title-list-item')
+    expect(title).toHaveValue('Butter')
   })
 
   it('toggles the checkbox on property change', () => {
-    const onToggleMock = jest.fn()
-    const onDeleteMock = jest.fn()
-    const { rerender, getByLabelText } = render(
-      <ListItem
-        isChecked={true}
-        title="Milk"
-        onToggle={onToggleMock}
-        onDelete={onDeleteMock}
-      />
-    )
-    const checkbox = getByLabelText('Milk')
+    const { rerender, getByTestId } = render(<ListItem isChecked={true} />)
+    const checkbox = getByTestId('list-item')
     expect(checkbox).toHaveProperty('checked', true)
-
-    rerender(
-      <ListItem isChecked={false} title="Milk" onToggle={onToggleMock} />
-    )
+    rerender(<ListItem isChecked={false} />)
     expect(checkbox).toHaveProperty('checked', false)
   })
 
-  it('calls onToggle', () => {
-    const onToggleMock = jest.fn()
-    const onDeleteMock = jest.fn()
-    const { getByText } = render(
-      <ListItem
-        onToggle={onToggleMock}
-        title="Bread"
-        isChecked
-        onDelete={onDeleteMock}
-      />
+  it('calls onToggleCheckbox', () => {
+    const onToggleCheckboxMock = jest.fn()
+    const { getByTestId } = render(
+      <ListItem onToggleCheckbox={onToggleCheckboxMock} />
     )
-
-    const listItem = getByText('Bread')
-    user.click(listItem)
-
-    expect(onToggleMock).toHaveBeenCalled()
+    const checkbox = getByTestId('checkbox')
+    user.click(checkbox)
+    expect(onToggleCheckboxMock).toHaveBeenCalled()
   })
 
   it('toggles the checkbox', () => {
-    const onToggleMock = jest.fn()
-    const onDeleteMock = jest.fn()
-    const { getByLabelText } = render(
-      <ListItem title="Milk" onToggle={onToggleMock} onDelete={onDeleteMock} />
+    const onToggleCheckboxMock = jest.fn()
+    const { getByRole } = render(
+      <ListItem onToggleCheckbox={onToggleCheckboxMock} />
     )
-    const checkbox = getByLabelText('Milk')
+    const checkbox = getByRole('checkbox')
+    // const checkbox = getByTestId('checkbox') not working
+    // material ui elment
     expect(checkbox).toHaveProperty('checked', false)
-
     user.click(checkbox)
-
     expect(checkbox).toHaveProperty('checked', true)
   })
+
   it('calls onDelete', () => {
-    const onToggleMock = jest.fn()
     const onDeleteMock = jest.fn()
-    const { getByTestId } = render(
-      <ListItem
-        onToggle={onToggleMock}
-        title="Bread"
-        isChecked
-        onDelete={onDeleteMock}
-      />
-    )
-
+    const { getByTestId } = render(<ListItem onDelete={onDeleteMock} />)
     const button = getByTestId('delete-list-item')
     user.click(button)
-
     expect(onDeleteMock).toHaveBeenCalledTimes(1)
   })
+
+  xit('calls handleChange', () => {
```

Comment:
Do you not want to execute this test? There is an "x" at the beginning of the line?